src/com/lftechnology/java/training/srijan/basics/coreapp/dao/DbConnection.java

```diff
@@ -0,0 +1,43 @@
+
+package com.lftechnology.java.training.srijan.basics.coreapp.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class DbConnection {
+
+	private static final Logger LOGGER =
+		Logger.getLogger(DbConnection.class.getName());
+
+	private DbConnection() {
+
+	}
+
+	/**
+	 * <p>Connects with database</p>
+	 * 
+	 * @return
+	 * @author srijan
+	 */
+	public static Connection dbConnection() {
+
+		String USER_NAME = "root";
+		String PASS = "liferay";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00117)